int64 numDisc
int64 contDim
float64[] contRangeLow
float64[] contRangeHigh


